src/main/java/org/jabref/model/pdf/FileAnnotationType.java

```diff
@@ -50,6 +56,26 @@ public static FileAnnotationType parse(PDAnnotation annotation) {
         }
     }
 
+    /**
+     * Determines if a String is a supported marked FileAnnotation type.
+     *
+     * @param annotationType a type descriptor
+     * @return true if annotationType is a supported marked FileAnnotation type
+     */
+    public static boolean isMarkedFileAnnotationType(String annotationType) {
+        for (FileAnnotationType type : Collections.unmodifiableList(Arrays.stream(FileAnnotationType.values())
+                .filter(FileAnnotationType::isLinkedAnnotationType).collect(toList()))) {
```

Comment:
An alternative and simpler solution would be to use Enum.valueOf... to parse the String. If it throws an Illegal Argument exception you simply return false...
Otherwise I would omit the loop and directly add a second filter Argument to the strea, and use anyMatch to return a boolean